app/src/main/java/fr/free/nrw/commons/explore/map/ExploreMapFragment.java

```diff
@@ -447,13 +447,15 @@ public void populatePlaces(LatLng curLatLng) {
             .observeOn(AndroidSchedulers.mainThread())
             .subscribe(explorePlacesInfo -> {
                     mediaList = explorePlacesInfo.mediaList;
+                    if(mediaList == null) {
+                        showErrorMessage(getString(R.string.no_pictures_in_this_area));
+                    }
                     updateMapMarkers(explorePlacesInfo);
                     lastMapFocus = new GeoPoint(curLatLng.getLatitude(), curLatLng.getLongitude());
                 },
                 throwable -> {
                     Timber.d(throwable);
-                    showErrorMessage(getString(R.string.error_fetching_nearby_places)
```

Comment:
I think this one should probably stay as the error for fetching nearby places - or we should understand what these errors are better, given I think this occurs e.g. when it tries to fetch places and the server is down, or there is no connection.

Reply:
Got it 👍,
I was thinking that too. Here we should display the actual message to the user like it was doing before, right?

Reply:
Yes I think so

Reply:
Okay :)
I am making the changes.